### ISSUE TITLE
Error and crash in word.

### ISSUE THREAD

**Issue by derekriemer** [nvaccess/nvda#5563]
If I can get word to not respond I will attempt to generate a dump. Here is a log snippet. I can provide the offending table if need be, but would prefer to email it since I don't have copyright to put it here publicly.

error executing script: <bound method Dynamic_IAccessibleWordDocumentWindowNVDAObject.script_nextRow of <NVDAObjects.Dynamic_IAccessibleWordDocumentWindowNVDAObject object at 0x04FF4C90>> with gesture u'ctrl+alt+down arrow'
Traceback (most recent call last):
  File "scriptHandler.pyc", line 186, in executeScript
  File "NVDAObjects\window\winword.pyc", line 1338, in script_nextRow
  File "NVDAObjects\window\winword.pyc", line 1332, in _moveInTable
  File "speech.pyc", line 760, in speakTextInfo
  File "textInfos\__init__.pyc", line 427, in getControlFieldSpeech
  File "speech.pyc", line 1083, in getControlFieldSpeech
  File "NVDAObjects\window\winword.pyc", line 580, in get
  File "NVDAObjects\window\winword.pyc", line 1034, in fetchAssociatedHeaderCellText
  File "comtypes\client\lazybind.pyc", line 163, in caller
  File "comtypes\automation.pyc", line 664, in _invoke
COMError: (-2147352567, 'Exception occurred.', (u'The requested member of the collection does not exist.', u'Microsoft Word', u'wdmain11.chm', 25421, None))

**Comment by jcsteh** [comment]
* What version of Word?
* How exactly do you reproduce this crash?
* Does it happen always, just sometimes or only in certain cases?
* How big is the table?

You can email the file to me directly. Thanks.

**Comment by derekriemer** [comment]
Hey Jamie:
I figured exactly how to reproduce.
This word doc does it.
The issue is with merged cells on the column level. Headers for row columns need to be enabled.
Create a word table in such a way that the first column has a set of headers of merged cells expanded over several vertical cells.
For example, row 1-3 col 1 is the catigory fish.
Columns 2 ... N are datapoints under the catigory of row n ... n+k
For example, to the right of fish, we have written in csv format:
point,value
point,value
...,...
point,value
Then we have a new catigory, say beef.
beef,point,value
,point,value
,point,value
,...
,point,value
where column 1 is one giant merged cell.
This word document is a table. This is not the table that I found in my bio lab book, so don't worry about copyright. This is just something I i whipped up to showcase this bug.
To reproduce:

* Open word and this file.
* Read the table without row headers enabled.
* Now enable row headers. Do this with NVDA Shift R on col1 row1.
* Note that Reading down by row works as expected in column 1.
* Go to col 2.
* Read row by row on col 2.

Expected: NVDA should speak the row header of the major category, i.e. Fish.
Observed: NVDA throws a COM error, and fails to get some index of some sort. I am not familiar with the code in question, or I would try to provide code details, but I suspect an enumeration error where NVDA fails to see this column category header as one cell.
[bug.docx](https://github.com/nvaccess/nvda/files/49546/bug.docx)
